sdcm/nemesis.py

```diff
@@ -459,9 +467,17 @@ def disrupt_hard_reboot_node(self):
         self.cluster.wait_for_nodes_up_and_normal(nodes=[self.target_node])
 
     def disrupt_multiple_hard_reboot_node(self):  # pylint: disable=invalid-name
+        # If this messages appeared, cdc successfully update generation
+        cdc_success_msg = ["streams description table already updated",
+                           "CDC description table successfully updated with generation"]
         num_of_reboots = random.randint(2, 10)
         for i in range(num_of_reboots):
-            self.log.info("Rebooting {} out of {} times".format(i + 1, num_of_reboots))
+            self._set_current_disruption('MultipleHardRebootNode %s' % self.target_node)
```

Comment:
With new Disruption event we shouldn't use _set_current_disruption inside the nemesis function. Please, use InfoEvent instead.

Reply:
fixed